app/src/main/java/turtler/voyageur/activities/TripActivity.java

```diff
@@ -24,7 +24,9 @@ public void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_trip);
         ButterKnife.bind(this);
 
-        TripFragmentPageAdapter pagerAdapter = new TripFragmentPageAdapter(getSupportFragmentManager());
+        Bundle b = new Bundle();
```

Comment:
you dont need to change tripactivity or tripfragmentpageadapter.

just do `String tripId = getActivity().getIntent().getStringExtra("tripId");` in your TripMapFragment